app/dao/services_dao.py

```diff
@@ -520,3 +520,25 @@ def dao_fetch_active_users_for_service(service_id):
     )
 
     return query.all()
+
+
+@statsd(namespace="dao")
+def dao_fetch_monthly_historical_stats_by_template():
+    month = get_london_month_from_utc_column(NotificationHistory.created_at)
+    year = get_london_year_from_utc_column(NotificationHistory.created_at)
+    end_date = datetime.combine(date.today(), time.min)
+
+    return db.session.query(
+        NotificationHistory.template_id,
+        month.label('month'),
+        year.label('year'),
+        func.count().label('count')
+    ).filter(
```

Comment:
Do we need to filter 'test' notifications? I know we weren't before but we are filtering those from the dashboard